kgcl/utils.py

```diff
@@ -96,7 +96,7 @@ def from_yaml(filename: str) -> Session:
 @click.argument("files", nargs=-1)
 def cli(files: List[str]):
     for f in files:
-        session = from_yaml(f)
+        session = from_yaml(f)  # TODO: Where is this 'session' var used?
```

Comment:
@ckindermann , does this `session` variable need to be used elsewhere?

Reply:
No, I don't think so. I am also quite confident that the function `cli` can be removed as well. I remember that we had some issues with the parsing and conversion functions. So, this was most likely only used to run some manual tests and inspect error messages.